edx_sga/static/js/src/edx_sga.js

```diff
@@ -100,6 +100,8 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                     }
                 }
             });
+
+            updateChangeEvent(fileUpload);
```

Comment:
Hi aliaksandrb,

Please Replace fileUpload with fileUploadObj, because of this like I am getting issues 

#### Javascript error
![screen shot 2015-06-05 at 4 54 35 pm](https://cloud.githubusercontent.com/assets/10431250/8005468/d528c6c8-0ba3-11e5-93e8-ae3bf403c3b2.png)

#### Second SGA is not loading properly 
![screen shot 2015-06-05 at 4 54 40 pm](https://cloud.githubusercontent.com/assets/10431250/8005487/1d733148-0ba4-11e5-94e7-15a342629942.png)

cc @pdpinch